internal/plugin/connectors/http/conjur/plugin.go

```diff
@@ -1,7 +1,9 @@
 package conjur
 
 import (
-	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"log"
+
+	generichttp "github.com/cyberark/secretless-broker/internal/plugin/connectors/http/generic"
```

Comment:
I had originally named the package `generichttp` but after feedback from Srdjan and (I thought) you  we decided to go with the name `generic` since we thought the "http" would be clear from context.

So I think either:
1. We revisit that decision and change the package.
2. Keep the package name and don't do the rename here.

At this point I'd lean toward 2.

Generally speaking, you should only be renaming a package because it has a conflict with another package.  Otherwise it's telling you that the package name is not good enough, and you should fix the root problem (assuming it's code we own).